scripts/2.mutation-classifier.py

```diff
@@ -181,11 +170,16 @@ def select_feature_set_columns(X, feature_set):
 
 # In[12]:
 
-
 # Parameter Sweep for Hyperparameters
-n_components_list = [50, 100]
-regularization_alpha_list = [10 ** x for x in range(-3, 1)]
-regularization_l1_ratio = 0.15
+regularization_alpha_list = [10 ** x for x in range(-10, 10)]
+# Chose n_components based on number of positives (or negatives, if that is less)
+n_positives = min(y.sum(),len(y)-y.sum())
```

Comment:
Style: spaces after comma and surrounding operators (`-`).

Let use a more accurate variable name, like `min_class_size`.